reliability/single-fit/trace/src/test/kotlin/com/datadog/android/trace/integration/opentracing/UtilitiesTest.kt

```diff
@@ -62,7 +63,8 @@ class UtilitiesTest {
 
     @AfterEach
     fun `tear down`() {
-        GlobalTracer::class.java.setStaticValue("isRegistered", false)
+        DatadogTracingToolkit.clear()
```

Comment:
I don't see such method at the `DatadogTracingToolkit` declaration. Is it extension method? upd: ok, seems some test extensions

Also the package for this test class is `com.datadog.android.trace.integration.opentracing` - should we update it/reorganize?

Reply:
1. Yes, moved all tests-related code into fixtures, so we could avoid reflection as much as possible 
2. Good notice, renamed